tools/init/lib/InitModule.ts

```diff
@@ -25,7 +25,7 @@ import { getModuleCommand } from "./commands/module";
 export default class InitModule extends Module {
   constructor() {
     // eslint-disable-next-line @typescript-eslint/no-var-requires,unicorn/prefer-module, @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access
-    super("init", require("../package.json").version);
+    super("init", require("../../package.json").version);
```

Comment:
Given the file path, this doesn't make sense. Would this not look in tools?

Reply:
Yes I agree, but because everything in lib ends up in ./dist/lib I need to go up twice.

Reply:
So from now on, we have to always do one path shorter if we go outside the lib?